### DISCUSSION TITLE
https://harrellbm.github.io/Quartz-Garden/

### DISCUSSION THREAD

**Discussion by giscus** [harrellbm/Quartz-Garden#2] [Announcements]
# https://harrellbm.github.io/Quartz-Garden/

Hello! 👋🏻 My name is Brenden Harrell. I am a Husband, Father, Lutheran Pastor, and Tinkerer. You have stubbled upon an experiment of mine. I have grown increasingly dissatisfied with the exhausting, overwhelming and overstimulating digital spaces we inhabit.

https://harrellbm.github.io/Quartz-Garden/

**Comment by PharmerPharper** [comment]
Stopped by the garden for the first time today. Excited to start strolling around!

**Reply by harrellbm** [reply]
I’m so glad you are checking it out! You get the prize for the first commenter 😜
